pkgs/by-name/ro/rocksndiamonds/package.nix

```diff
@@ -3,35 +3,44 @@
   stdenv,
   fetchurl,
   makeDesktopItem,
+  copyDesktopItems,
   SDL2,
   SDL2_image,
   SDL2_mixer,
   SDL2_net,
   zlib,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "rocksndiamonds";
   version = "4.4.0.4";
 
   src = fetchurl {
-    url = "https://www.artsoft.org/RELEASES/linux/${pname}/${pname}-${version}-linux.tar.gz";
+    url = "https://www.artsoft.org/RELEASES/linux/${finalAttrs.pname}/${finalAttrs.pname}-${finalAttrs.version}-linux.tar.gz";
```

Comment:
use of pname like this is discouraged. May as well get rid of it while we're at it